### PR TITLE
Update Italian translation

### PR DESCRIPTION
**Pull request by auanasgheps**

### CODE REVIEW

**Comment by AdrienPoupa** [comment]
Thanks!

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -206,16 +239,22 @@
...
+    <string name="some_files_are_not_listed_in_the_media_store">Alcuni file non sono elencati nel media store.</string>
     <string name="not_listed_in_media_store"><![CDATA[<b>%s</b> non è elencato nel media store.]]></string>
     <string name="nothing_to_scan">Nulla da scansionare.</string>
     <string name="scanned_files">Scansione effettuata per %1$d su %2$d file.</string>
     <string name="could_not_scan_files">Non è stato possibile effettuare la scansione di %d file.</string>
-    <string name="listing_files">Creazione elenco dei file</string>
+    <string name="listing_files">Creazione elenco dei brani...</string>
+    <string name="scanning_x_songs_in_progress">Aggiornamento di %1$d brani…</string>
+    <string name="scanning_x_songs_finished">Aggiornati %1$d brani</string>
+    <string name="listing_files">Lettura dei file</string>
```
Comment (soncaokim) [inline app/src/main/res/values-it/strings.xml:257]: This is creating a build error since `listing_files` is duplicated

Comment (auanasgheps) [reply app/src/main/res/values-it/strings.xml:257]: Sorry. Do you want me to send a new PR?

Comment (soncaokim) [reply app/src/main/res/values-it/strings.xml:257]: Yes please

Comment (auanasgheps) [reply app/src/main/res/values-it/strings.xml:257]: Done #477

**Pull request merged**